modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -255,7 +255,9 @@ def update_status!(status:, code: nil, detail: nil)
             from: current_status,
```

Comment:
(Github won't allow me to comment above this line, but I'm referring to line **251** here):

This keeps the previous behavior of only saving a status update if the status (e.g. `error` vs. `success`) has changed, and does not take into account the `code` or `detail`. This means we wouldn't get a status update entry if we got two successive `error` statuses that had different `code`s and `detail`s. I'm still not very familiar with the types of error codes and details that might come up - should I update this to check code and detail as well?